examples/otlpmetric/src/main/java/com/google/cloud/opentelemetry/example/otlpmetric/OTLPMetricExample.java

```diff
@@ -74,6 +77,10 @@ private static MetricExporter addAuthorizationHeaders(
                 .toBuilder()
                     .addHeader(
                         "Authorization", "Bearer " + credentials.getAccessToken().getTokenValue());
+
+        if (credentials.getQuotaProjectId() != null) {
```

Comment:
Does the Authorization header work with refresh as is?

Reply:
No, the refresh will not work in this sample yet - this sample still needs to be updated to use the auth extension which would handle this.
Additionally, the sample in itself is correct - the use case shown here does not require refresh.

This change is made to reduce confusion around the quota headers and adding GCP resource detection. I'm working on adding metrics support in the auth extension, after which this sample would require another update.